Enable thumb build to reduce build size of u-boot.kwb (Closes: #897671).

Without this, u-boot.kwb overlaps where the u-boot environment is
stored, and updating the environment can break u-boot and vice versa.

Index: u-boot/configs/sheevaplug_defconfig
===================================================================
--- u-boot.orig/configs/sheevaplug_defconfig
+++ u-boot/configs/sheevaplug_defconfig
@@ -33,3 +33,4 @@ CONFIG_USB_EHCI_HCD=y
 CONFIG_USB_STORAGE=y
 CONFIG_LZMA=y
 CONFIG_OF_LIBFDT=y
+CONFIG_SYS_THUMB_BUILD=y
